pkg/collector/py/utils.go

```diff
@@ -452,3 +453,31 @@ func GetPythonIntegrationList() ([]string, error) {
 
 	return ddPythonPackages, nil
 }
+
+// SetPythonPsutilProcPath sets python psutil.PROCFS_PATH
+func SetPythonPsutilProcPath(procPath string) error {
+	glock := newStickyLock()
+	defer glock.unlock()
+
+	ns := strings.Split(pyPsutilProcPath, ".")
+	pyPsutilModule := ns[0]
+	psutilModule := python.PyImport_ImportModule(pyPsutilModule)
+	if psutilModule == nil {
```

Comment:
I know we don't do it either in the other bindings, but I think that when this is `nil` then the exception flag is set in the python runtime (see https://docs.python.org/2/c-api/import.html#c.PyImport_ImportModule), so we should call `glock.getPythonError()` to grab the error and reset the flag. On top of that, it'll give us a more accurate error.